### PR TITLE
more tests for better coverage

### LINKED ISSUE #491: Unit tests coverage for properties helpers
Some classes in the `com.crowdin.cli.properties` namespace have low coverage by Unit tests. It would be great to add Unit tests for these classes.

- [ ] [PropertiesBuilders.java](https://app.codecov.io/gh/crowdin/crowdin-cli/blob/cli3/src/main/java/com/crowdin/cli/properties/PropertiesBuilders.java)
- [ ] [BasePropertiesBuilder.java](https://app.codecov.io/gh/crowdin/crowdin-cli/blob/cli3/src/main/java/com/crowdin/cli/properties/BasePropertiesBuilder.java)
- [ ] [PropertiesWithTargets.java](https://app.codecov.io/gh/crowdin/crowdin-cli/blob/cli3/src/main/java/com/crowdin/cli/properties/PropertiesWithTargets.java)
- [ ] [ProjectPropertiesBuilder.java](https://app.codecov.io/gh/crowdin/crowdin-cli/blob/cli3/src/main/java/com/crowdin/cli/properties/ProjectPropertiesBuilder.java)
- [ ] [PropertiesBuilderChecker.java](https://app.codecov.io/gh/crowdin/crowdin-cli/blob/cli3/src/main/java/com/crowdin/cli/properties/PropertiesBuilderChecker.java)
- [ ] [PropertiesWithTargetsBuilder.java](https://app.codecov.io/gh/crowdin/crowdin-cli/blob/cli3/src/main/java/com/crowdin/cli/properties/PropertiesWithTargetsBuilder.java)

Similar tests for reference can be found [here](https://github.com/crowdin/crowdin-cli/tree/cli3/src/test/java/com/crowdin/cli/properties)

### PR DESCRIPTION
**Pull request by calfaand**

### CODE REVIEW

**Comment by andrii-bodnar** [comment]
Hi @calfaand, thanks a lot for your contribution! 🚀 

It's strange but it seems like the new tests are not affecting the coverage (according to the [codecov comment](https://github.com/crowdin/crowdin-cli/pull/509#issuecomment-1325542902))

Could you please take a look?

File: src/test/java/com/crowdin/cli/commands/functionality/PropertiesBuilderTest.java
Comment (andrii-bodnar) [inline src/test/java/com/crowdin/cli/commands/functionality/PropertiesBuilderTest.java:261]: add a newline above

File: src/test/java/com/crowdin/cli/commands/functionality/PropertiesBuilderTest.java
Comment (andrii-bodnar) [inline src/test/java/com/crowdin/cli/commands/functionality/PropertiesBuilderTest.java]:
```suggestion
    public void testMockPropBuildersWithTargets() {
```

Comment (andrii-bodnar) [reply src/test/java/com/crowdin/cli/commands/functionality/PropertiesBuilderTest.java]: and please do the same for other new tests

**Comment by calfaand** [comment]
Yes it seems strange to me because I verified it in Intelijj idea and it shows different coverage, but I take look at it later afternoon

**Review by andrii-bodnar — APPROVED** [review]

**Comment by andrii-bodnar** [comment]
@calfaand thank you! I'm going to merge this PR and then will see if the coverage will be increased.

**Comment by andrii-bodnar** [comment]
@calfaand the coverage wasn't changed after the merge.

But I found another interesting thing:

The `PropertiesBuilderTest.java` class is located in the `java/com/crowdin/cli/commands/functionality` namespace,

but the `PropertiesBuilders.java` class itself is located in the `java/com/crowdin/cli/properties` namespace.

It could be the reason for not accounting coverage. Could you please move the test class to the right namespace?

**Pull request merged**
